controllers/src/test/scala/cromwell/pipeline/controller/ProjectControllerTest.scala

```diff
@@ -34,6 +35,16 @@ class ProjectControllerTest extends AsyncWordSpec with Matchers with ScalatestRo
           responseAs[Project] shouldEqual getProjectByNameResponse
         }
       }
+
+      "fail fail with access denied if wrong user" taggedAs Controller in {
```

Comment:
```suggestion
      "return status code InternalServerError if service returned failure" taggedAs Controller in {
```
This test doesn't check anything about user. It only checks that we return `InternalServerError` if service returns `Future.failed`

Reply:
Ok, maybe i need to add some check about response body answer where we have a message about access denied ?

Reply:
It depends on what you really want to test. `fail with access denied if wrong user` doesn't say anything because I have no idea why this user is "wrong" :)
In case of `access denied` event we return 403 (Forbidden) code, not 500. So I just suggested test name based on its implementation.

Reply:
Ok, thanks. Now I see.